### PR TITLE
Detail popup window

### PR DESCRIPTION
**Pull request by gabrielcoder247**
### `PopUp Window`
**In this milestone**
I and Debas, my partner.
Added the following.

> Html (I added template for popup window)

> Css (I added styles for the popup window)

> Javascript(Debas added Open and close function for the popUp window and I added the evenListener method)

### CODE REVIEW

**Review by mke2111 — CHANGES_REQUESTED** [review]
### Hey @gabrielcoder247, 

**This is a great project you have going on, well-done lad**. 

   
**However, there are a few sections to improve before your project is approved.** 🔴 
<hr>

## Required Changes ♻️
_Check the inline comments under the review for sections to improve._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 1]
```diff
@@ -48,6 +48,48 @@
...
+                <a class="btn-live" id="btn-live" href="#" target="_blank">See Live <img id="l-img" src="images/ic_link_teal.png"
+                    alt="see live icon"></a>
+                <br />
+                <a class="btn-source" id="btn-source" href="#" target="_blank">See Source <img id="s-img" src="images/ic_github.png"
+                    alt="see source icon"></a>
+            </div>
+            <div class="btn-window" id="d-btn-window">
+                <a class="btn-live" id="d-btn-live" rel="noopener" href="https://gabrielcoder247.github.io/" target="_blank">See Live <img id="l-img-btn" src="images/ic_link_teal.png"
+                    alt="see live icon"></a>
+                <br />
```
Comment (mke2111) [inline index.html]: - [ ] The modal popup should be populated from the array of objects containing all your projects. Delete this code from the HTML structure.

File: index.html [hunk 2]
```diff
@@ -202,7 +149,7 @@ <h3 class="title ">Project name goes here...</h3>
                     <div class="rails language-title ">Ruby/Rails</div>
                     <div class="js language-title ">Javascript</div>
                 </div>
-                <a class="see-this-project " href="# ">
+                <a class="see-this-project" id="see-this-project-1" href="#">
                     <p class="text ">see this projects</p>
                     <img class="arrow-d " src="images/Union.png " alt="icon-arrow ">
```
Comment (mke2111) [inline index.html]: - [ ] Delete this code pertaining to the projects from the HTML file. Everything related to the project should be populated from the array of objects within the javascript file.

File: popup.js [hunk 3]
```diff
@@ -0,0 +1,202 @@
...
+  tech: ['CSS', 'HTML', 'Bootstrap', 'Ruby'],
+},
+{
+  id: 3,
+  name: 'Project name goes here...',
+  image: 'images/Rectangle-21devices.png',
+  description: 'This is project that i have done in microverse ',
+  tech: ['CSS', 'HTML', 'Bootstrap', 'Ruby'],
+},
+{
```
Comment (mke2111) [inline popup.js]: - [ ] You are currently having your projects in `projectData` and `projectObj`. It is better to stick to the array of objects i.e. `projectData` to store your project details. It should be from this that you populate all your projects and the modal popup in the HTML file to display your projects. You are using `projectData` for your projects and `projectObj` for part of the modal popup

File: popup.js [hunk 4]
```diff
@@ -0,0 +1,202 @@
...
+  const projTech = document.querySelectorAll('#l-items li');
+  projTech.forEach((item, i) => {
+    item.textContent = projectObj[projId].tech[i];
+  });
+
+  const projLive = document.querySelector('#btn-live');
+  projLive.href = projectObj[projId].live;
+
+  const projSource = document.querySelector('#btn-source');
+  projSource.href = projectObj[projId].source;
```
Comment (mke2111) [inline popup.js]: - [ ] As pointed out in the first comment, populate the modal popup from the array of objects i.e `projectData`.

**Review by SpaYco — COMMENTED** [review]
Hello 👋,

This is an additional review as a result of the second opinion request.

Populating the page with javascript is part of the project requirement

Check [here](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/8_popup.md) 👇

> - In order to associate each project with the popup window details, you must use a JavaScript array to store all of the information for all projects.
> 
>   - For each project you need to store the following pieces of data in a JavaScript object, at a minimum: name, description, featured image, technologies, link to live version, link to source.
>   - You need to update the main page so that the projects section is created dynamically using the information stored in that JavaScript object. Remember, all of the HTML in that section is created when the page loads.
>   - After that, you could implement the popup window.

Please do the changes then ask for another review

Keep rocking 🚀!

**Review by SajjadAhmad14 — APPROVED** [review]
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:

### Hi Gabriel Nwachukwu, 
## Congratulations 🎉 Thank you for implementing changes suggested previously 🙏

## Happy Coding 👏👏👏
#### Please leave any questions or comments in the PR if you have any doubts or unclarity!
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**